From: Jason Abele <jason.abele@gmail.com>
Date: Wed, 11 Nov 2015 17:51:30 -0800
Subject: LOCAL: fix my ubuntu x86 builds

---
 Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile b/Makefile
index f1437d2..eb58663 100644
--- a/Makefile
+++ b/Makefile
@@ -13,7 +13,6 @@ EXTRA_CFLAGS += -Wno-unused-label
 EXTRA_CFLAGS += -Wno-unused-parameter
 EXTRA_CFLAGS += -Wno-unused-function
 EXTRA_CFLAGS += -Wno-unused
-EXTRA_CFLAGS += -Wno-error=date-time
 
 #EXTRA_CFLAGS += -Wno-uninitialized
 
