From: Jason Abele <jason.abele@gmail.com>
Date: Thu, 12 Nov 2015 16:07:52 -0800
Subject: rtl8723bs_mp_driver: enable adaptivity for ETSI/CE mark

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f1437d2..7cbb813 100644
--- a/Makefile
+++ b/Makefile
@@ -51,7 +51,7 @@ CONFIG_TRAFFIC_PROTECT = y
 CONFIG_LOAD_PHY_PARA_FROM_FILE = y
 CONFIG_CALIBRATE_TX_POWER_BY_REGULATORY = n
 CONFIG_CALIBRATE_TX_POWER_TO_MAX = n
-CONFIG_RTW_ADAPTIVITY_EN = disable
+CONFIG_RTW_ADAPTIVITY_EN = enable
 CONFIG_RTW_ADAPTIVITY_MODE = normal
 CONFIG_SKIP_SIGNAL_SCALE_MAPPING = n
 CONFIG_80211W = n
